packages/pigeon/e2e_tests/test_objc/ios/.gitignore

```diff
@@ -31,3 +31,6 @@ Runner/GeneratedPluginRegistrant.*
 !default.mode2v3
 !default.pbxuser
 !default.perspectivev3
+
+Runner/dartle.h
```

Comment:
Add a comment as to why these are excluded?

Reply:
Done